test/firmata.test.js

```diff
@@ -515,6 +515,51 @@ describe("board", function() {
     serialPort.emit("data", [ANALOG_MESSAGE | (1 & 0xF), 0 % 128, 0 >> 7]);
   });
 
+  it("should be able to read value of analog pin on a board that skipped capabilities check", function(done) {
+    var serialPort = new SerialPort("/path/to/fake/usb");
+    var board = new Board(serialPort, {skipCapabilities: true, analogPins: [14,15,16,17,18,19]}, function(err) {});
```

Comment:
Actually the test would fail because of [this line](https://github.com/jgautier/firmata/blob/master/lib/firmata.js#L1258). However this could be fixed by tracking the analog pin reporting values and digital pin reporting values in separate arrays. Maybe this is a separate pull request though. It's worth rethinking how analog and digital pins are tracked now that a need for something like skipCapabilities has been established.